### PR TITLE
Label breaking changes

### PR DESCRIPTION
**Pull request by wenovus**
See manual testing at https://github.com/openconfig/public/pull/779

### CODE REVIEW

File: commonci/github.go [hunk 1]
```diff
@@ -249,13 +249,18 @@ func (g *GithubRequestHandler) AddPRComment(body *string, owner, repo string, pr
...
 // signature, then the comment is edited. If none matches, then a new comment
 // is posted.
-func (g *GithubRequestHandler) AddOrEditPRComment(signature string, body *string, owner, repo string, prNumber int) error {
+// If body is nil, then it indicates delete.
+func (g *GithubRequestHandler) AddEditOrDeletePRComment(signature string, body *string, owner, repo string, prNumber int) error {
 	if signature == "" {
+		if body == nil {
+			return fmt.Errorf("PR comment body unspecified")
+		}
 		return g.AddPRComment(body, owner, repo, prNumber)
```
Comment (wenovus) [inline commonci/github.go:264]: This is not related to the rest of the changes. Just adding additional functionality to the function.

**Review by bstoll — APPROVED** [review]

**Comment by wenovus** [comment]
Thanks for the review!
